app/models/event_type.py

```diff
@@ -0,0 +1,43 @@
+import uuid
+from app.models import db
+from app.helpers.helpers import get_count
+
+
+def get_new_slug(name):
+    slug = name.lower().replace(" ", "-")
+    count = get_count(EventType.query.filter_by(slug=slug))
+    if count == 0:
+        return slug
+    else:
+        return '{}-{}'.format(slug, uuid.uuid4().hex)
+
+class EventType(db.Model):
+    """Event type object table"""
+
+    __tablename__ = 'event_types'
+
+    id = db.Column(db.Integer, primary_key=True, unique=True, autoincrement=True)
+    name = db.Column(db.String, nullable=False)
+    slug = db.Column(db.String, unique=True)
+    events = db.relationship('Event', backref="event_types")
```

Comment:
And shouldn't the back reference be `event_type` ? Since one event can have only one type